config/routes.rb

```diff
@@ -58,6 +58,8 @@
 
   controller :orders do
     get :cart
+    get :order_success
+    get :order_cancel
```

Comment:
Any action that is making any change in the db shouldn't be a get request. Get request should be only for fetching records

Reply:
Discussed